core-bundle/src/Resources/contao/templates/backend/be_tinyMCE.html5

```diff
@@ -64,6 +64,10 @@ window.tinymce && tinymce.init({
     contextmenu: false,
   <?php $this->endblock(); ?>
 
+  <?php $this->block('cache_suffix'); ?>
+    cache_suffix: '?v=<?= System::getContainer()->get('assets.packages')->getVersion('js/tinymce.min.js', 'contao-components/tinymce4') ?>',
```

Comment:
I don‘t like having to use `System::getContainer()` in templates. @m-vo How would we handle this in Twig?

Reply:
@leofeyer Symfony provies an `asset_version` Twig method in its Twig Extension. We could also add a `Template::assetVersion` helper method, alongside `Template::asset`.

Reply:
Yes, please add an `assetVersion()` method. 👍